setup.cfg

```diff
@@ -35,7 +35,7 @@ install_requires =
     datashader
     holoviews
     ipycytoscape >=1.0.3
-    ipywidgets
+    ipywidgets >=7.6,<8
```

Comment:
looks like your `ipyctyoscape` pin need updating :point_up: pyviz_comms, probably also needs an update... their stuff i think that pin is there to _avoid_ pulling the lab3 one

Reply:
Can I just remove the pyviz_comms dependency/pin altogether for now?

Reply:
dunno... not sure if it comes along with hv/ds... you'd have to check _their_ install_requires. If it's jupyter-only (possible) they probably won't, and you would want it.